.circleci/config.yml

```diff
@@ -484,6 +484,7 @@ jobs:
           name: Admin User Setup
           command: |
             if [ "${CIRCLE_BRANCH}" != "prod" ]; then
+              . ./scripts/load-environment-from-secrets.sh
```

Comment:
Do we still need to source the vars from secrets? I know this was part of our debugging effort but I may have lost context at some point!

Reply:
yeah, I think at one point I mentioned we might be able to remove those, but I'm pretty sure that fixed an issue where it kept trying to create the admin user with admin@example.com when the secrets expected it to create ustcadmin@example.com.  Without this change the build creates the wrong admin user name.

Reply:
Oh that's right - thank you for explaining it to me again!